### PR TITLE
form ts [feature]

### PR DESCRIPTION
**Pull request by Sakai-san**

### CODE REVIEW

File: pyrene/src/components/Form/Form.tsx [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+import React from 'react';
+import { ValidationError, ObjectSchema } from 'yup';
+import { MultiselectOption, Filters, Options } from '../Filter/types';
+
+type ValuesOf<T> = T[keyof T];
+type KeysOf<T> = keyof T;
+
+type FormValues = Filters;
+type ToucheValues = {
```
Comment (maximelovino) [inline pyrene/src/components/Form/Form.tsx:14]: I would rename this type to `TouchedValues`

File: pyrene/src/components/Form/Form.tsx [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+type InputComponentProps = {
+  name: KeysOf<FormState['values']>,
+  value: ValuesOf<FormState['values']>,
+  invalid: boolean,
+  invalidLabel: string,
+  onChange: (value: string, event: React.ChangeEvent<any>) => void,
+  onBlur: (event: React.FocusEvent<any>) => void,
+};
+
+const getTouchedState = (initialValues: FormValues) => (
```
Comment (maximelovino) [inline pyrene/src/components/Form/Form.tsx:52]: Since this is only for computing the initial state, I would rename it to `getInitialTouchedState`

**Review by maximelovino — CHANGES_REQUESTED** [review]
The type and the function should be renamed imo

File: pyrene/src/components/Form/Form.tsx (hunk 2 above)
Comment (Sakai-san) [reply pyrene/src/components/Form/Form.tsx:52]: @maximelovino Ok. I ll do so. This task was originally only to do the migration. But I can also do the renaming.

File: pyrene/src/components/Form/Form.tsx (hunk 1 above)
Comment (Sakai-san) [reply pyrene/src/components/Form/Form.tsx]: > I would rename this type to TouchedValues
Thanks. Nice catch !

**Review by maximelovino — APPROVED** [review]

**Pull request merged**